### ISSUE TITLE
Could/should firmware be placed on a GPT disk outside the useable LBAs

### ISSUE THREAD

**Issue by daniel-thompson** [ARM-software/ebbr#21]
On systems where the boot ROM is hardcoded to fetch firmware from a block address >=2 and <34 (yes, they exist) it appears possible and standards compliant to place (for example):

* Protective MBR at LBA0
* GPT header in LBA1
* Firmware at LBA16
* GPT partition table in LBA(8192 - 32)
* First usable LBA: LBA8192

Current EBBR text is unclear about whether the above is permitted or not. Currently we require GPT except where it is impossible *and* we require firmware to reside within protective partitions except where impossible. It's not clear which of these rules should take precedent!

**Comment by wmamills** [comment]
I think such firmware is protected by the MBR and seems protected by the GPT.  The fact that it is not in a "partition" is not relevant.

However, partition editing tools may break this.  This should be tested with several  partition tools.

**Comment by glikely** [comment]
I think it would be worthwhile to revisit the shared block device scheme, but I don't think we've got any actions against EBBR at the moment. I'm going to close this issue until we've got a concrete way forward.

I've been discussing a possible better approach with Steve McIntyre that should make it easier to support the distros, but we need to get it written up into something that can be posted to the boot-architecture mailing list.
